test/JDBC/expected/SELECT_INTO_TEST-vu-verify.out

```diff
@@ -0,0 +1,304 @@
+create table select_into(select_into_COL int);
+go
+select select_into_COL from select_into;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table select_into_REPO with column select_into_COL1 using function
+select select_into_double_function(select_into_COL) as select_into_COL1 into select_into_REPO from select_into;
+go
+-- Indexing over column select_into_COL1
+create index IDX_REPRODUCTION on select_into_REPO(select_into_COl1);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute join pg_class on pg_attribute.attrelid=pg_class.oid where pg_class.relname='select_into_repo' and attname like '%select_into%'order by attname asc;
+go
+~~START~~
+varchar#!#varchar
+select_into_col1#!#{bbf_original_name=select_into_COL1}
+~~END~~
+
+-- dropping the table select_into
+drop table if exists select_into;
+go
+-- dropping the table select_into_REPO
+drop table if exists select_into_REPO;
+go
+drop function if exists select_into_double_function;
+go
+
+-- create a new table select_into 7
+create table select_into([select_into_COL$] int);
+go
+select [select_into_COL$] from select_into; 
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table select_into_REPO with column select_into_COL 
+select [select_into_COL$] as [select_into_COL1$] into select_into_REPO from select_into;
+go
+-- Indexing over column select_into_COL
+create index IDX_REPRODUCTION on select_into_REPO([select_into_COL1$]);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute join pg_class on pg_attribute.attrelid=pg_class.oid where pg_class.relname='select_into_repo' and attname like '%select_into%'order by attname asc;
+go
+~~START~~
+varchar#!#varchar
+select_into_col1$#!#{bbf_original_name=select_into_COL1$}
+~~END~~
+
+-- dropping the table select_into
+drop table if exists select_into;
+go
+-- dropping the table select_into_REPO
+drop table if exists select_into_REPO;
+go
+
+--dependent objects
+create index IDX_REPRODUCTION on select_into_pre_exist_repro(select_into_pre_exist_COL);
+go
+drop table if exists select_into_pre_exist;
+go
+drop table if exists select_into_pre_exist_repro;
+go
+-- column length >=64 todo
+create table select_into(select_into_COL_select_into_COL_select_into_COL_select_into_COL_ int);
+go
+select select_into_COL_select_into_COL_select_into_COL_select_into_COL_ from select_into;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table select_into_REPO with column select_into_COL 
+select select_into_COL_select_into_COL_select_into_COL_select_into_COL_ into select_into_REPO from select_into;
+go
+-- Indexing over column select_into_COL
+create index IDX_REPRODUCTION on select_into_REPO(select_into_COL_select_into_COL_select_into_COL_select_into_COL_);
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: column "select_into_col_select_into_colb68a72f6d624cf699ece5bf8f6446ce4" does not exist)~~
```

Comment:
This shouldn't be failing.

Reply:
here the column name length > 63, so we are not lowercasing the column name for this case and not storing the original name

Reply:
Done